### DISCUSSION TITLE
Suggestion to add all hours like in Nordpool (if possible)?

### DISCUSSION THREAD

**Discussion by Flotteemse** [MTrab/stromligning#26] [Ideas]
Hi
This is my first GitHub idea post, so sorry in advance if something is not as it should be.
I have installed the custom component and it seems to be running fine. 
When I look at the sensors availble from the integration I get: "Current", "Maximum", "Mean" and "Minimum", which are all nice to have. But would it be possible to get all hours of the day like in the Nordpool integration? Or is the data just not available?
Thanks for your contribution anyway :)

**Comment by MTrab** [comment]
They are already there.
Today are in the current sensor and tomorrow are in the tomorrow available sensor

**Reply by Flotteemse** [reply]
Ahh yes, sorry. Found it now!

And if others get to this post - it can be found here: Developer tools -> States -> Filter entities (search: stromligning).
Then you will see that the info is available in the "attributes" column like below:
![image](https://github.com/user-attachments/assets/d34fc149-776b-41e7-a784-bc25a022d9ff)
